plugins/sentinelone/help.md

```diff
@@ -17,6 +17,11 @@ This plugin utilizes the SentinelOne API, the documentation is located in the Se
 
 * Sentinel one API administrative credentials
 
+# Supported Product Versions
+
+* 2.0.0
+* 2.1.0
```

Comment:
Do we need both versions? Not only the newer one?

Reply:
No we need two because 2.0.0 have v1 api and 2.1.0 v2 api